app-misc/kryoflux-dtc/metadata.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<email>ck+gentoo@bl4ckb0x.de</email>
+		<name>Conrad Kostecki</name>
+	</maintainer>
+	<maintainer type="project">
+		<email>proxy-maint@gentoo.org</email>
+		<name>Proxy Maintainers</name>
+	</maintainer>
+	<use>
+		<flag name="demos">Install example floppy game files.</flag>
+		<flag name="fast-firmware">Install fast firmware instead of slow firmware.</flag>
+		<flag name="java">Install the Kryoflux GUI</flag>
```

Comment:
Again, I'm sorry that it didn't occur to me earlier but please make it USE=gui.

Reply:
Done!